### DISCUSSION TITLE
看评论里说710事件,到底是个啥?

### DISCUSSION THREAD

**Discussion by jonny77** [WeNeedHome/SummaryOfLoanSuspension#616] [General]
有没有图

**Comment by Mmmmeruem** [comment]
twitter搜 河南村镇银行

**Reply by ZJuntao** [reply]
既然都上推特了，直接搜郑州岂不更快

**Reply by dinglino1** [reply]
> 既然都上推特了，直接搜郑州岂不更快
你能看到什么新闻不是你决定的，由我党决定的。

**Comment by TALK1949** [comment]
https://chinadigitaltimes.net/chinese/684319.html
不知道你能不能打开，里面汇总了不少视频和观点

**Reply by mutoto321** [reply]
https://youtu.be/9j4X6Ra4D00这个视频很全看了寒心

**Comment by yalo-ddaa** [comment]
被殴打受伤流血了

**Comment by C-Joey** [comment]
有事先Google啊

**Comment by jiushi555** [comment]
基层干部和群众打成一片

**Reply by 1418478505** [reply]
体现了政府体察民情，和人民水乳交融的和谐场景

**Reply by hclgdb** [reply]
“白衣战神”！

**Comment by tanuk1997** [comment]
别尬黑，善意镇压

**Reply by sqpk** [reply]
恶意要钱是吧

**Comment by Universe-Wong** [comment]
打成一片

**Comment by xiaohao587** [comment]
都到这儿还不知道710事件 ？

**Reply by zhw173** [reply]
这里有时候不要梯子

**Comment by pengyidong** [comment]
在人民银行面前，人民政府秉着为人民服务的宗旨，安排人民警察发动人民战争，与人民群众打成一片

**Comment by 505Games-oss** [comment]
习近平:别看你今天闹得欢，当心啊，日后拉清单。

**Reply by xijinping0** [reply]
嘿嘿嘿

[![拉清单](https://img.youtube.com/vi/iBg0UhhV72E/0.jpg)](https://youtu.be/iBg0UhhV72E?t=113)

https://github.com/xijinping0

**Comment by b4o** [comment]
希望成立一个组织，对贪污的人全球追杀，为什么西方政府要给这些人渣畜生提供庇护，为什么不直接枪毙？

**Reply by Cegazhos** [reply]
前提是高度全球化，或者全球共产主义社会。现在最多就是给国际刑警发通缉令 帮忙抓罢了。
至于为什么不枪毙，那就是司法上的事了

**Comment by hclgdb** [comment]
现在要加上918了

**Reply by Cegazhos** [reply]
> 现在要加上918了

918什么事?是某大学停电那就算了吧
